src/test/java/OstrichTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * This file contains sample JUnit test cases for Ostrich.java
+ */
+
+import org.junit.*;
+
+import static org.junit.Assert.*;
+
+
+public class OstrichTest {
```

Comment:
Need test cases for downgrade speed and toString.

Reply:
test case for downgrade speed added.